game/src/engine/asset/AssetService.cpp

```diff
@@ -0,0 +1,395 @@
+#include "engine/core/debug/Log.h"
+#include "engine/asset/AssetService.h"
+#include "engine/scene/Entity.h"
+
+using namespace std;
+
+void AssetService::LoadModel(const string &path, const string &name)
+{
+    // Model importing
+    Assimp::Importer importer;
+
+    // Process the desired calculation for the model
+    unsigned int flag;
+    flag = aiProcess_Triangulate |              // Transform all the model's primitive
+                                                // shapes to triangles
+           aiProcess_JoinIdenticalVertices |    // Identifies and joins identical
+                                                // vertex data sets within all
+                                                // imported meshes
+           aiProcess_CalcTangentSpace |         // Calculate the tangents and
+                                                // bitangents for the imported meshes
+           aiProcess_GenNormals |               // Create normal vectors for each vertex if
+                                                // the model doesn't contain normal vectors
+           aiProcess_MakeLeftHanded |           // Convert all the imported data to a
```

Comment:
I think these two should be removed. OpenGL uses a right-handed coordinate system, and counter-clockwise winding order by default